Week-4/assignment4_1.js

```diff
@@ -0,0 +1,15 @@
+const getNumber = () =>{
+    new Promise((res,rej) => {
+        const randomNum = parseInt(Math.random() * 100 , 10);
+        setTimeout(() => {
+            if (randomNum % 5 === 0) {
+                rej(`Rejected with num ${randomNum}`)
+            }
+            res(`Resolved with num ${randomNum}`)
+            
+        }, randomNum * 10)
+    })
+}
+
+const x = getNumber()
+x.then()
```

Comment:
X will now have pending prmosie , now next you have to either handle it or catch the error like...
x.then(()=>).catch(()=>)